src/tunacode/tools/glob.py

```
@@ -147,6 +147,120 @@ def _expand_brace_pattern(pattern: str) -> list[str]:
+            compiled.append((pat, re.compile(regex_pat, flags)))
+        else:
+            compiled.append((pat, re.compile(fnmatch.translate(pat), flags)))
+    return compiled
+
+
+def _single_pattern_matches(
+    entry_name: str,
+    rel_path: str,
+    orig: str,
+    comp: re.Pattern[str],
+    recursive: bool,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)